pkg/hyperkube/README.md

```diff
@@ -0,0 +1,15 @@
+This package contains files copied from the main [kubernetes/kubernetes repo](https://github.com/kubernetes/kubernetes).
```

Comment:
Is there an upstream issue yet to move these out to another repo?

Reply:
https://github.com/kubernetes/kubernetes/issues/54299